### PR TITLE
DOCS-2315: updates log rotate tutorial with new steps && new behaviors

### PR DESCRIPTION
**Pull request by schmalliso**

### CODE REVIEW

File: source/includes/steps-log-rotate-rename.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+stepnum: 1
+ref: start-mongod
+title: "Start a :program:`mongod` instance, specifying the {{logrotateOption}} :option:`--logRotate` behavior."
+replacement:
+  logrotateOption: "``rename``"
+action:
+  language: sh
+  code: |
+    mongod -v --logpath /var/log/mongodb/server1.log --logRotate reopen
```
Comment (kay-kim) [inline source/includes/steps-log-rotate-rename.yaml]: did you mean --logRotate rename here?

Comment (schmalliso) [inline source/includes/steps-log-rotate-rename.yaml]: Yes. Yes I did.

Comment (kay-kim) [inline source/includes/steps-log-rotate-rename.yaml]: : )

File: source/tutorial/rotate-log-files.txt [hunk 2]
```diff
@@ -7,111 +7,69 @@ Rotate Log Files
...
-
 .. seealso:: For information on logging, see the
    :ref:`monitoring-standard-loggging` section.
 
-Log Rotation With MongoDB
--------------------------
+Log Rotation with ``--logRotate rename``
+----------------------------------------
 
 The following steps create and rotate a log file:
```
Comment (kay-kim) [inline source/tutorial/rotate-log-files.txt]: I know you blurbed above in the overview, but would also be nice to have the descriptions within the sections as well.

Comment (schmalliso) [inline source/tutorial/rotate-log-files.txt]: I agree! Updated :)

File: source/tutorial/rotate-log-files.txt [hunk 3]
```diff
@@ -7,111 +7,76 @@ Rotate Log Files
...
+UTC timestamp to the filename, in :term:`ISODate` format. It then
 opens a new log file, closes the old log file, and sends all new log
 entries to the new log file.
 
-MongoDB's standard approach to log rotation only rotates logs
-in response to the :dbcommand:`logRotate` command, or when the
-:program:`mongod` or :program:`mongos` process receives a ``SIGUSR1``
-signal from the operating system.
+With the :option:`--syslog` option, :program:`mongod` and :program:`mongos`
+report all activity to the ``syslog`` rather than a log file.
```
Comment (kay-kim) [inline source/tutorial/rotate-log-files.txt]: I think you meant this to go below.  See where you have the "Finally, you can configure .. ``syslog``.

Comment (schmalliso) [inline source/tutorial/rotate-log-files.txt]: I think I actually meant to get rid of the Finally… paragraph, but I think I'll go with your suggestion. Jeez.

File: source/includes/steps-log-rotate-reopen.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+stepnum: 1
+title: "Start a :program:`mongod` instance, specifying the ``reopen`` :option:`--logRotate` behavior."
```
Comment (kay-kim) [inline source/includes/steps-log-rotate-reopen.yaml]: this won't build because no  ref in yaml

Comment (schmalliso) [inline source/includes/steps-log-rotate-reopen.yaml]: indeed. jeez.

File: source/tutorial/rotate-log-files.txt [hunk 5]
```diff
@@ -7,111 +7,74 @@ Rotate Log Files
 Overview
 --------
 
-Log rotation using MongoDB's standard approach archives the current
+When used with the :option:`--logpath` option or :setting:`systemLog.logPath` setting,
```
Comment (kay-kim) [inline source/tutorial/rotate-log-files.txt]: almost -- the setting I believe is :setting:`systemLog.path`

**Comment by kay-kim** [comment]
merged : )

**Pull request closed without merging**